monkey/infection_monkey/utils/plugins/plugin_test.py

```diff
@@ -0,0 +1,31 @@
+from unittest import TestCase
+
+from infection_monkey.utils.plugins.pluginTests.PluginWorking import pluginWorking
+from infection_monkey.utils.plugins.pluginTests.BadImport import SomeDummyPlugin
+from infection_monkey.utils.plugins.pluginTests.BadInit import badPluginInit
+from infection_monkey.utils.plugins.pluginTests.PluginTestClass import TestPlugin
+
+
+class PluginTester(TestCase):
```

Comment:
Awesome. Can you add a test case for a file which has one class of each, just to really be sure of the robustness? (i.e. one bad class, one bad init, one good class).